pipelined/bevy_render2/src/render_resource/uniform_vec.rs

```diff
@@ -86,6 +90,9 @@ impl<T: AsStd140> UniformVec<T> {
         }
         self.reserve(self.values.len(), device);
         if let Some(uniform_buffer) = &self.uniform_buffer {
+            if self.item_size * self.values.len() == 128 {
+                panic!("ARGH");
```

Comment:
This could have a more descriptive error message :smile:

Reply:
🤣 Indeed. I thought I'd removed all hacky panics but I guess one slipped through. :)

Reply:
This was actually some debug code that should outright not be there. Removed.